.github/workflows/ci-build.yml

```diff
@@ -26,6 +26,26 @@ jobs:
         python-version: '3.6'
     - name: ${{ matrix.scripts }}
       run: bash qa/${{ matrix.scripts }}.sh;
+  dependency-analysis:
```

Comment:
Keep this in a separate file.

Reply:
Actually @sawood14012 wanted it to be in this file only.